src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -107,6 +107,7 @@ class GetUrlsTest : public CommandRequestTest<CommandsTestMocks::kIsNice> {
 
 TEST_F(GetUrlsTest, RUN_SUCCESS) {
   EXPECT_CALL(mock_policy_handler_, PolicyEnabled()).WillOnce(Return(true));
+  EXPECT_CALL(app_mngr_, GetRPCService()).WillOnce(ReturnRef(rpc_service_));
```

Comment:
@BSolonenko please replace with ON_CALL (everywhere)
Try to avoid using EXPECT_CALLS for non critical for use cases getters.

Reply:
@LitvinenkoIra  Fixed in https://github.com/LitvinenkoIra/sdl_core/pull/4/commits/976a6c79a2ae3df1626a91f07de3e0f80f8ec788